syncserver-settings/src/lib.rs

```diff
@@ -176,7 +176,7 @@ impl Default for Settings {
             statsd_host: Some("localhost".to_owned()),
             statsd_port: 8125,
             human_logs: false,
-            cors_allowed_origin: None,
```

Comment:
Ah, yeah, I think actix changed this default a while ago. `None` now means None, not all.

Sorry, should have caught that.